.jenkins/continuous.groovy

```diff
@@ -343,6 +343,8 @@ pipeline {
                         dockerfile {
                             filename "Dockerfile.sycl"
                             dir "docker"
+                            // FIXME_KOKKOS_VERSION replace by 3.7 once released
+                            additionalBuildArgs '--build-arg KOKKOS_VERSION="22ee6208ffe2b04881e9e59338a636e97e9c9ae5"'
```

Comment:
Why don't you change it in https://github.com/arborx/ArborX/blob/master/docker/Dockerfile.sycl?

Reply:
I find it confusing if the Dockerfile actually provides a (valid) default value if we are expecting to change/override it in the `.jenkins` file. At the moment, we have
```
# FIXME hash below correspond to 3.5.99 (pre 3.6 release develop branch)
ARG KOKKOS_VERSION=bb340781ba774eb0829de03735fe25e89b1176fc
```
in `Dockerfile.sycl` so I would think that comment is up-to-date.

Reply:
I too find it confusing. Lets discuss this at today's meeting so that we can get some common understanding.